### PR TITLE
Inhands can now come in ANY SIZE, not just 32x32!

### PR DESCRIPTION
**Pull request by RemieRichards**
## Feature:
- Items may now specify dimensions for their `lefthand_file` and `righthand_file` variables (which point to their inhand files), these dimensions allow for INHANDS OF ANY SIZE FROM 1x1 TO INFINITYxINFINITY!
## Code:
- Adds `center_image(image, x_dimension, y_dimension)` which centers an image based on given values for x and y dimensions, this is to allow larger inhands to be drawn as if the human occupied the center of the larger inhand file (giving the spriter the maximum about of bonus pixels in each direction)

Crude example:
I_am_not_a_spriter

### CODE REVIEW

**Comment by Anonus** [comment]
durand leg inhand when

**Comment by RemieRichards** [comment]
@Anonus 
draw it, that's when

**Comment by PKPenguin321** [comment]
giant anime swords when

**Comment by RemieRichards** [comment]
@PKPenguin321
draw it, that's when, hand them to an admin, they'll be able to VV any item to use files of any size, including your giant swords

**Comment by PKPenguin321** [comment]
i'm ready

**Comment by WJohn** [comment]
Oh hey I was just thinking about these for facehugger sprites for large queens.

Nice timing!

**Comment by GunHog** [comment]
Oh, RR. You have done it now. :+1:

**Comment by Fox-McCloud** [comment]
Remie, Remie why!?

**Comment by Thunder12345** [comment]
Are the two different colours on the blade in the example part of the sprite or a lighting thing?

**Comment by RemieRichards** [comment]
@Fox-McCloud *grumble grumble* pixels *grumble grumble*
@Thunder12345 lighting.

**Comment by Thunder12345** [comment]
Is there any way to stop it happening, or do we just have to live with it?

**Comment by Miauw62** [comment]
nope, there's nothing you can do about that.

**Comment by RemieRichards** [comment]
@Thunder12345 besides changing how we do lighting (lol no) it's not really possible.
This shouldn't occur much anyways, only where the inhand escapes from a lit tile to an unlit one (like in the picture)

**Comment by Miauw62** [comment]
isnt that more or less the point of having inhands bigger than 32x32 though? :P

**Comment by RemieRichards** [comment]
@Miauw62 yes, but you're not always pushed up against a wall for it to trigger this issue :P

**Comment by Miauw62** [comment]
fair enough

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1448,3 +1448,40 @@ B --><-- A
 	if(orbiting)
 		loc = get_turf(orbiting)
 		orbiting = null
+
+
+//Center's an image.
```
Comment (Miauw62) [inline code/__HELPERS/unsorted.dm]: should be centers, not center's. :V

Comment (RemieRichards) [inline code/__HELPERS/unsorted.dm]: Criticise not my english, I'm lazy, and this like this are ingrained errors from long ago.
I sometimes randomly capitalise words too, that's another one.

tl;dr shush :S

Comment (GunHog) [inline code/__HELPERS/unsorted.dm]: You also add useless "u"s to things, such as "armour". 'MERICUH!!

Comment (RemieRichards) [inline code/__HELPERS/unsorted.dm]: That is NOT an error, that is the correct form of British English, we simply use two different languages/variants.

Comment (GunHog) [inline code/__HELPERS/unsorted.dm]: I added the " 'MERICUH!! " there for a reason! HONK!!

Comment (Xhuis) [inline code/__HELPERS/unsorted.dm]: :us: :us: :boom: :gun: :man: :boom: :us: :us:

**Comment by RemieRichards** [comment]
@tgstation/commit-access 

Merge this, y'know, for the badmins!
